python/pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
 dependencies = [
     "aiohttp ~= 3.8",
     "cloudevents ~=1.0",
-    "pydantic >=2.0,<2.10",
+    "pydantic >=2.0,<2.11,!=2.10.0,!=2.10.1,!=2.10.2,!=2.10.3",
```

Comment:
Could we just do `>=2.10.4`?

Reply:
No, then everything stops working with earlier versions then 2.10